code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -396,6 +396,11 @@ Contains:
 	if (user.head == src)
 		user.faction -= "carp"
 
+/obj/item/clothing/suit/space/hardsuit/carp/old
```

Comment:
This is basically the traitor hardsuit considering slowdown doesn't matter in space. I'd nerf the armour values since space protection is already pretty strong

Reply:
Oh, that wasn't intentional, I'll go give it a nerf.